src/FsAutoComplete/LspHelpers.fs

```diff
@@ -517,6 +517,10 @@ module ClassificationUtils =
 
 type PlainNotification= { Content: string }
 
+type TestDetectedNotification =
```

Comment:
@baronfel, do we want to keep this type here or move it to another file?

Reply:
it's fine to stay here. At some point I'd like to move the custom API response structures to a dedicated file that we can point folks to for integration purposes, but that's all just window-dressing.